bundle/src/main/java/com/adobe/acs/commons/functions/Function.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2010, 2013, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
```

Comment:
pretty sure we can't include this. Isn't there a Function class in Guava which can be reused...